lib/internal/modules/esm/worker.js

```diff
@@ -35,17 +28,7 @@ const { isMarkedAsUntransferable } = require('internal/buffer');
  */
 function transferArrayBuffer(hasError, source) {
   if (hasError || source == null) { return; }
-  let arrayBuffer;
-  if (isArrayBuffer(source)) {
-    arrayBuffer = source;
-  } else if (isTypedArray(source)) {
-    arrayBuffer = TypedArrayPrototypeGetBuffer(source);
-  } else if (isDataView(source)) {
-    arrayBuffer = DataViewPrototypeGetBuffer(source);
-  }
-  if (arrayBuffer && !isMarkedAsUntransferable(arrayBuffer)) {
-    return [arrayBuffer];
-  }
+  return ArrayBufferPrototypeTransfer(source);
```

Comment:
Even though this method is named as "transferArrayBuffer", it does not behave as `ArrayBufferPrototypeTransfer`.

Reply:
Yes you're right. Thank you for the review.